Index: g15mpd-1.0.0/Makefile.am
===================================================================
--- g15mpd-1.0.0.orig/Makefile.am
+++ g15mpd-1.0.0/Makefile.am
@@ -6,7 +6,7 @@ bin_PROGRAMS = g15mpd
 docdir = $(prefix)/share/doc/$(PACKAGE)-$(VERSION)
 doc_DATA = README ChangeLog AUTHORS NEWS COPYING
 
-LIBS = -lg15daemon_client -lg15render -lX11 -lXtst -lmpd 
+LIBS = -lg15daemon_client -lg15render -lX11 -lXtst -lmpd -lpthread
 IMCLUDES = -I
 g15mpd_LDFLAGS = -avoid-version
 g15mpd_SOURCES =  g15mpd.c
